setup.py

```diff
@@ -51,7 +51,7 @@
     "pyyaml>=5.1.0",
     "requests>=2.0.0",
     "tqdm>=4.0.0",
-    "pydantic>=1.8.2,<2.0.0",
```

Comment:
Please delete : local/features/pydantic-upgrade/v2_validation.py

Reply:
Oof good catch Damian, I thought this folder was in my local .gitignore